integration-tests/bats/helper/local-remote.bash

```diff
@@ -136,6 +136,7 @@ SKIP_SERVER_TESTS=$(cat <<-EOM
 ~cli-hosted.bats~
 ~profile.bats~
 ~ls.bats~
+~rebase.bats~
```

Comment:
What happens if we run the `rebase.bats` tests when there's a local sql-server running already? I see that we have a separate bats test file for local-remote mode, but I wanted to understand if there's some functionality that doesn't work in local-remote mode. Is it possible to get the same test file running for both local-remote and local modes?

Reply:
The problem is that the rebase tests use `dolt checkout` for setup which won't work remotely :(

Reply:
Ahh... yes... I think that was the same issue last time I asked about this list, too. Thanks for gently reminding me 😉 

Bummer that we keep hitting that snag. I can't think of a great way to avoid having to use `dolt checkout` for the tests. The only idea that comes to mind is using `main` as the branch to rebase, since that happens to already be the default branch for CLI-mode. You could temporarily switch to another branch and make commits if you sent a batch of SQL statements to `dolt sql -q`, but I don't think it's worth it to rewrite the tests like that.